dbt_copilot_helper/utils/files.py

```diff
@@ -44,3 +44,31 @@ def ensure_cwd_is_repo_root():
             bg="red",
         )
         exit(1)
+
+
+def generate_override_files(base_path, file_path, output_dir):
+    for file in file_path.glob("*"):
```

Comment:
These two bits of code are identical apart from the glob.

Do we need the two globs because we want all the files in the root of the directory and all the files from the `bin` directory?

We could perhaps extract the duplicated code to a helper method and just pass in the glob.

Reply:
Yes, just the root and bin directories. I did have a go at refactoring this by trying to do it all in one glob pattern, but then it required excepting some other directories.